From 659f89e2c7ad33f966b133f853f873bc1121ead6 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.xyz>
Date: Wed, 12 Jun 2024 15:32:46 +0800
Subject: [PATCH 127/133] loongarch(loongarch.cc): drop incomplete
 TARGET_OPTION_RESTORE implementation

This was backported to the GCC 12 maintenance branch (?!) but is broken.

Signed-off-by: Mingcong Bai <baimingcong@uniontech.com>
---
 gcc/config/loongarch/loongarch.cc | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/gcc/config/loongarch/loongarch.cc b/gcc/config/loongarch/loongarch.cc
index 3eacf59bbbb..d30d5dd7468 100644
--- a/src/gcc/config/loongarch/loongarch.cc
+++ b/src/gcc/config/loongarch/loongarch.cc
@@ -7269,20 +7269,6 @@ loongarch_option_save (struct cl_target_option *,
   loongarch_update_gcc_opt_status (&la_target, opts, opts_set);
 }
 
-/* Implement TARGET_OPTION_RESTORE.  */
-static void
-loongarch_option_restore (struct gcc_options *,
-			  struct gcc_options *,
-			  struct cl_target_option *ptr)
-{
-  la_target.cpu_arch = ptr->x_la_opt_cpu_arch;
-  la_target.cpu_tune = ptr->x_la_opt_cpu_tune;
-
-  la_target.isa.fpu = ptr->x_la_opt_fpu;
-
-  la_target.cmodel = ptr->x_la_opt_cmodel;
-}
-
 /* Implement TARGET_CONDITIONAL_REGISTER_USAGE.  */
 
 static void
-- 
2.43.4

